### DISCUSSION TITLE
FusionDarkStyle dark theme

### DISCUSSION THREAD

**Discussion by ProfessorUdGuru** [pawelsalawa/sqlitestudio#4125] [Q&A]
In the Styles folder there's something called FusionDarkStyle.dll and I've seen a screenshot of the dark theme. I don't see a way to change to dark theme in the menus. Thanks.

**Comment by pawelsalawa** [comment]
![image](https://user-images.githubusercontent.com/481846/119729139-9928be00-be74-11eb-99dd-4aa92d072fb3.png)

**Reply by ProfessorUdGuru** [reply]
Thanks. Looks good.
